src/packer.pkr.hcl

```diff
@@ -136,19 +143,57 @@ build {
   }
 
   provisioner "windows-restart" {
-    # Wait a maximum of 30 minutes for Windows to restart.
-    # The build will fail if the restart process takes longer than 30 minutes.
-    restart_timeout = "30m"
+    # Wait a maximum of 5 minutes for Windows to restart. The build will fail
+    # if the restart process takes longer than 5 minutes.
+    restart_timeout = "5m"
+  }
+
+  provisioner "powershell" {
+    # Wait 30 seconds before executing the next provisioner. This gives a grace
+    # period between restarting Windows and running the next provisioner.
+    # Disable Windows Defender, enable and configure RDP, set the desktop
+    # wallpaper, and install Chocolatey and Boxstarter.
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/set-wallpaper.ps1",
+      "src/powershell/install-chocolatey.ps1",
+      "src/powershell/install-boxstarter.ps1"
+    ]
+  }
+
+  provisioner "powershell" {
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir C:\\${var.packages_dir}"]
+  }
+
+  provisioner "file" {
+    # Upload package lists to the "packages" directory.
+    destination = "C:\\${var.packages_dir}"
+    source      = "src/${var.packages_dir}/"
+  }
+
+  provisioner "windows-restart" {
```

Comment:
Would this be more appropriate before the previous two provisioners that just move files? Not a major deal but it seems "best" to restart right after the bigger setup provisioner and just let the two file transfer provisioners run right before using the uploaded files.

Reply:
@mcdonnnj I don't think this will make a difference in the behavior, so I went ahead and updated to your preference in [889c7d6](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/889c7d61e730fe42bdd82ca7146673c741f7fb81)

Reply:
@itsmostafa I have updated https://github.com/cisagov/windows-commando-vm-packer/pull/21#discussion_r1327627577 to use this commit as you made both changes in the commit you linked.